vmm/src/memory_manager.rs

```diff
@@ -1504,7 +1507,7 @@ impl MemoryManager {
         for r in &self.guest_ram_mappings {
             let vm_dirty_bitmap = self
                 .vm
-                .get_dirty_log(r.slot, r.gpa, r.size, 0)
+                .get_dirty_log(r.slot, r.gpa, r.size, DIRTY_BITMAP_CLEAR_DIRTY)
```

Comment:
I am assuming `mshv` will get and reset the dirty-page bitmap with `DIRTY_BITMAP_CLEAR_DIRTY`, right? 

It ensures that there is no racing and no potential missing dirty-page-tracking.

Reply:
Yes. It ensures that.